extension/js/common/client-configuration.ts

```diff
@@ -110,6 +112,21 @@ export class ClientConfiguration {
     return this.clientConfigurationJson.enforce_keygen_expire_months;
   };
 
+  /**
+   * An array of strings to check against the subject of the composed password-protected message.
+   * If any string in this array is found in the subject, an error alert must be displayed.
+   */
+  public getDisallowPasswordMessagesForTerms = (): string[] | undefined => {
+    return this.clientConfigurationJson.disallow_password_messages_for_terms;
```

Comment:
let's also filter non-empty strings here, to avoid some unexpected behavior (as noticed by Den at https://github.com/FlowCrypt/flowcrypt-android/issues/2905#issuecomment-2528260144)